README.md

```diff
@@ -308,6 +327,9 @@ Unique document ID
 `doc.data` _(Object)_
 Document contents. Available after document is fetched or subscribed to.
 
+`doc.presence` _(Object)_
+Each property under `doc.presence` contains presence data shared by a client subscribed to this document. The property name is an empty string for this client's data and connection IDs for other clients' data. The structure of the presence object is defined by the OT type of the document (for example, in [ot-rich-text](https://github.com/Teamwork/ot-rich-text#presence) and [@datavis-tech/json0](https://github.com/datavis-tech/json0#presence)).
```

Comment:
I'm not sure I agree with handing this off to the types, because:

- I'm not sure it's strictly part of "OT", and it feels like a violation of Single Responsibility
- This relies on us being able to update the type libraries (can we?)
- There are potentially multiple desired implementations for a given type. Especially if you consider - for example - `json0`, which can sub-type other arbitrary types. What is a "generic" shape for this presence?

To me, it feels more natural for us to define some sort of separate presence transformers, probably being passed in to the `presence` constructor option?